### PR TITLE
pre-push git-hook for tsc

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Create a git hook that performs `tsc` on all commits 

## Why?

To enforce that the code compile, at least, as TC does it automatically.

### CODE REVIEW

**Comment by shtukas** [comment]
Thanks @nicl !

File: scripts/git-hooks/pre-push [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
```
Comment (SiAdcock) [inline scripts/git-hooks/pre-push]: This comment can go ♻️

**Comment by SiAdcock** [comment]
Nice one, thanks @shtukas!

File: scripts/git-hooks/pre-push (hunk 1 above)
Comment (shtukas) [reply scripts/git-hooks/pre-push]: Nice one. Thanks Simon!
https://github.com/guardian/dotcom-rendering/pull/344

**Pull request merged**